cetmix_tower_webhook/readme/USAGE.md

```
@@ -0,0 +1,3 @@
+When a request is received, Cetmix Tower will search for the webhook with the matching endpoint and authenticate the request using the selected authenticator. In case of successful authentication webhook code is run. Each webhook call is logged. Logs are available under the  "Cetmix Tower > Logs > Webhook Calls" menu or under the "Logs" button directly in the Webhook.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)